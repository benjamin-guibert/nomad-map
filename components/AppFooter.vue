<template>
  <footer class="footer my-colors-light shadow text-right mt-4 py-1">
    <div class="container">
      <p class="d-inline mx-1">Benjamin Guibert</p>
      <b-link
        href="https://github.com/benjamin-guibert"
        target="_blank"
        title="GitHub"
      >
        <font-awesome-icon class="mx-1" :icon="['fab', 'github']" size="lg" />
      </b-link>
    </div>
  </footer>
</template>
