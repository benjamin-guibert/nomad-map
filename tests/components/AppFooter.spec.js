import { createLocalVue, mount } from '@vue/test-utils'
import BootstrapVue from 'bootstrap-vue'
import AppFooter from '@/components/AppFooter.vue'

const localVue = createLocalVue()
localVue.use(BootstrapVue)

describe('AppFooter', () => {
  const wrapper = mount(AppFooter, {
    localVue,
    stubs: {
      FontAwesomeIcon: true
    }
  })

  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
