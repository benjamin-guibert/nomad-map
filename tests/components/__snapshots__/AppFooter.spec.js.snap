// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter renders 1`] = `"<footer class=\\"footer my-colors-light shadow text-right mt-4 py-1\\"><div class=\\"container\\"><p class=\\"d-inline mx-1\\">Benjamin Guibert</p> <a title=\\"GitHub\\" rel=\\"noopener\\" target=\\"_blank\\" href=\\"https://github.com/benjamin-guibert\\" class=\\"\\"><fontawesomeicon-stub icon=\\"fab,github\\" size=\\"lg\\" class=\\"mx-1\\"></fontawesomeicon-stub></a></div></footer>"`;
